README.md

```diff
@@ -42,7 +42,10 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> RSR | GSG | BSB | R | G | B
+R -> "r"
+G -> "g"
+B -> "b"
```

Comment:
This is correct. For me, I find it clear to express it as

S -> r | g | b | rSr | gSg | bSb

The reason is that it R is a nonterminal, then it can potentially take different values if it appears twice in a production. That's not the case here, but it was still a little confusing when I first read it.